genesis.json

```diff
@@ -16,5 +16,6 @@
         {"0x0000000000000000000000000000000000000000": {"balance": 0}},
         {"0x0000000000000000000000000000000000000000": {"balance": 0}},
         {"0x0000000000000000000000000000000000000000": {"balance": 0}}
-    ]
+    ],
+    "state": []
```

Comment:
Please, check [genesis library](https://github.com/WingsDao/griffin-consensus-poc/blob/delegates/lib/genesis.js#L31-L38) for compatibility with this change.